src/bfs/bfs_kernel.hpp

```diff
@@ -84,7 +79,7 @@ uint16_t run_bfs(const size_t num_vertices,
     /// BFS loop for a single level
     /// We assume that the cost of generating threads at every level is negligible
 #ifdef _OPENMP
-#pragma omp parallel for schedule (runtime)
+#pragma omp parallel for
```

Comment:
Does this default to static schedule? If so, could you add a comment to that effect?